packages/ui/lib/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import Footer from './Footer'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+export default {
+  component: Footer,
+  title: 'Footer',
+} as ComponentMeta<typeof Footer>
+
+const Template: ComponentStory<typeof Footer> = (args) => <Footer {...args} />
+
+export const Normal = Template.bind({})
+
+Normal.args = {
+  subscriptionForm: {
+    // example of implementation when form is submitted
+    onSubmit: async (email: string) => {
+      const endpoint = `https://api.hsforms.com/submissions/v3/integration/submit/19942922/868101be-ae3e-422e-bc86-356c96939187`
+      const options = {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          formGuid: '19942922',
+          portalId: '868101be-ae3e-422e-bc86-356c96939187',
+          fields: [
+            {
+              name: 'email',
+              value: email,
+            },
+          ],
+        }),
+      }
+
+      await fetch(endpoint, options)
```

Comment:
don't need this implementation in storybook. will end up causing false submissions.